sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/AlertTriggeringLiveTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class AlertTriggeringLiveTests : MetricsAdvisorLiveTestBase
+    {
+        public AlertTriggeringLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
+        [TestCase("anomalyTime")]
+        [TestCase("createdTime")]
+        [TestCase("modifiedTime")]
+        public async Task GetAlerts(string timeModeName)
+        {
+            MetricsAdvisorClient client = GetMetricsAdvisorClient();
+
+            AlertQueryTimeMode timeMode = timeModeName switch
+            {
+                "anomalyTime" => AlertQueryTimeMode.AnomalyTime,
+                "createdTime" => AlertQueryTimeMode.CreatedTime,
+                "modifiedTime" => AlertQueryTimeMode.ModifiedTime,
+                _ => throw new ArgumentOutOfRangeException("Invalid test case.")
+            };
+
+            var options = new GetAlertsOptions(SamplingStartTime, SamplingEndTime, timeMode);
+
+            var alertCount = 0;
+
+            await foreach (AnomalyAlert alert in client.GetAlertsAsync(AlertConfigurationId, options))
+            {
+                Assert.That(alert, Is.Not.Null);
```

Comment:
what is the difference between doing this and `Assert.IsNotNull(alert)`?

Reply:
It's the same thing. I personally prefer the `Assert.That(...)` pattern because of its readability.

Reply:
it makes it hard to read :( I understand when there is a customized error message though but not sure of the rest of scenarios

Reply:
```cs
Assert.That(..., Is.False);
Assert.That(..., Is.Empty);
Assert.That(..., Throws.InstanceOf<ArgumentException>());
...
```

Reply:
Well... I think it's a matter of personal opinion at this point. I don't think it's worth changing it so close to code complete date. If you feel strongly about it, I can open an issue to update the pattern in MA.

Reply:
Personally I find it easier to read because it sounds pretty much like English:
"Assert that `variable` is empty"
"Assert that `function` throws instance of Argument Exception"
...

But I don't feel animosity towards the other approach.

Reply:
Nah not strong opinion about it. I was just curious if there was another advantage or something I was missing